src/main/java/org/cyclops/integratedterminals/client/gui/container/ContainerScreenTerminalStorage.java

```diff
@@ -612,6 +609,18 @@ public Slot getSlotUnderMouse() {
         return slot;
     }
 
+    public boolean isSlotUnderMouseCraftingResult() {
```

Comment:
Instead of adding a hard-coupling to the itemstackcrafting-tab, can we make this more generic, so that other tab types could override this as well in the future?

We could for instance add a method `isQuickMovePrevented()` here and in the `ITerminalStorageTabClient` interface, which can then be implemented using the logic below in the itemstackcrafting-tab of `ITerminalStorageTabClient`.